### ISSUE TITLE
Can Detectron be installed in win10?

### ISSUE THREAD

**Issue by Kasdision** [facebookresearch/Detectron#276]
I have installed the caffe2-gpu successfully in my computer(win10). Now what I want is installing detectron in win10. Can this be done?
I haven't see any instruction in installing detectron in win10. Can anyone help? Thanks a lot in advance.

**Comment by gadcam** [comment]
You managed to install Caffe2 you should already be a wizard! 
More seriously, Detectron does not seem to support Windows currently see #25.
Maybe someone could help you if you try to install it with make and share the output.

**Comment by Kasdision** [comment]
@gadcam Oh, it's so sad to hear that. So I need try to install it in ubuntu16.04. Thank you anyway~

**Comment by ir413** [comment]
Hi @Kasdision - as noted by @gadcam and in @25, we don't support building on Windows at this time. However, if you were able to build Caffe2 on Windows successfully I wouldn't expect setting up Detectron to require much more work on top of that.

**Comment by MiZhangWhuer** [comment]
Hi @Kasdision @gadcam @ir413  , I have successfully complied caffe2 & Detectron under win7 platform, everything including python interface testing is OK except for multi-gpu inference. I tested the test_net.py on single GPU and it works, but when employing multi-gpu, it finally failed. I debug the scripts through PyCharm, and it indicates that some errors occurs on function p = subprocess.Popen in utils/subprocess.py.
After checking this function, I found that there isn't CUDA_VISIABLE_DEVICES envrionment variable in windows platform. So I comment this line " subprocess_env['CUDA_VISIBLE_DEVICES'] = str(gpu_ind)" in process_in_parallel function, but new issues occurs "AssertionError: Range subprocess failed (exit code: 1)
" (See snapshot)
![rgb8rwrodmhqxo _extfqt1](https://user-images.githubusercontent.com/18107150/47204662-6e218800-d3b6-11e8-9743-d72b5eac32db.png)

How can I fix this error on windows platform? It seems that this error related to mutli-thread programming on python.

Our windows version link: https://github.com/MiZhangWhuer/Detectron-windows

Hope further communication.
